srcpkgs/python3-editables/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'python3-editables'
+pkgname=python3-editables
+version=0.3
+revision=1
+wrksrc="${pkgname/python3-/}-${version}"
+build_style=python3-pep517
+hostmakedepends="python3-setuptools python3-wheel"
+depends="python3"
+checkdepends="python3-pip python3-coverage python3-pytest-cov python3-pytest python3-virtualenv"
```

Comment:
why does this need pip and virtualenv?

Reply:
I'm pretty sure there was a reason I added this, but I don't know anymore, and the tests of `editables` seem to pass without it, so I've removed them again.